z_infra_provisioning/cloud_infra_center/ocp_upi/roles/configure-installer-client/tasks/main.yaml

```diff
@@ -57,11 +55,9 @@
   when: openshift_install_download_results.changed
 
 - name: Download openshift client
-  become: yes
   get_url:
     url: "{{ openshift_client_url }}/openshift-client-linux.tar.gz"
     dest: "{{ openshift_install_dir }}/openshift-client-linux.tar.gz"
-    mode: 0755
```

Comment:
what is the mode of installer file when removed 0755?

Reply:
We don't need 0755 as system umask as default is 0022, so in the root user the image download will be as 0755 and in the non-root user the image will belong to user.  If put 0755 in it , it need to use sudo to handle the file.